.github/workflows/windows-msvc.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: actions/cache@v1
+      id: cache
```

Comment:
How do we update the cache if a newer gtk is needed?

Reply:
by changing the cache key to e.g gtk4-8